pkgs/top-level/python-aliases.nix

```
@@ -158,6 +158,7 @@ mapAliases ({
   email_validator = email-validator; # added 2022-06-22
   enhancements = throw "enhancements is unmaintained upstream and has therefore been removed"; # added 2023-10-27
   et_xmlfile = et-xmlfile; # added 2023-10-16
+  etebase-server = throw "pkgs.python3.etebase-server has been removed, use pkgs.etebase-server"; # added 2024-07-16
```

```suggestion
  etebase-server = pkgs.etebase-server; # added 2024-07-16
```
that should also just work

---

You could end up with the wrong python version used, see this PRs description.